ng2-components/ng2-alfresco-upload/src/components/upload-drag-area.component.ts

```diff
@@ -78,14 +79,15 @@ export class UploadDragAreaComponent {
     onUploadFiles(e: CustomEvent) {
         e.stopPropagation();
```

Comment:
I know was not introduced by your PR but can we remove this one character variable name ?